### DISCUSSION TITLE
v1.8.5 - Bug Fixes

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#901] [General]
* Fixed driver not installing on some systems
* Fixed Super Presets not auto switching correctly on Output device change
* Fixed CPU usage on some PlugIn Effects that have a small UI size
* Fixed app not opening on first try when there's an update available

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.8.5'>v1.8.5 - Bug Fixes</a>.</em>

**Comment by angusgastle** [comment]
This update appears to have broken the app on my M1 Mac. Audio flips on/off repeatedly and is now un-usable.

**Comment by angusgastle** [comment]
Totally broken now. :-(
